.rubocop.yml

```diff
@@ -50,12 +54,21 @@ Style/CollectionMethods:
 Style/WordArray:
   Enabled: false
 
+Style/SymbolArray:
+  Enabled: false
+
 Style/RegexpLiteral:
   Enabled: false
 
 Style/StringLiterals:
   Enabled: false
 
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
We are using ruby < 2.3 in production, so it seemed better to keep the explicit `freeze`s. Maybe I should add a comment stating that.